src/Components/Home/questions.json

```diff
@@ -0,0 +1,5 @@
+[
+  "How often do you think you are going to use this product?",
+  "Do you have free space to store this product when you are not using it?",
+  "Is this product you are considering buying safe for the environment? Why?"
+]
```

Comment:
I think the second question would provoke more reflection if ask "Where do you have free space...?"  The third one is really two questions, let's make it simpler and also make it open-ended: "How can buying this product be harmful to environment?" I feel like "yes/no" answers doesn't really make a user to think over the problem.

We can also add "which alternative products have you considered?"